src/frame/js/services/core.js

```diff
@@ -1,8 +1,6 @@
 import { Smooch } from 'smooch-core/lib/smooch';
 import urljoin from 'urljoin';
 
-import { VERSION } from '../../../shared/js/constants/version';
```

Comment:
VERSION is now injected directly by webpack... not sure why it wasn't that way in the first place (I think it was because npm lib was just babelified and not processed by webpack at all, now it is).